data-raw/process_data_sets_starry.R

```diff
@@ -9,11 +10,160 @@ colnames(cabinet_turnover) <- colnames(cabinet_turnover) %>%
+
+devtools::use_data(governor_national_forecast, overwrite = TRUE)
+devtools::use_data(governor_state_forecast, overwrite = TRUE)
+
+# house-forecast-2018 ---------------------------------------------------------------
+
+house_district_forecast <- read_csv("https://projects.fivethirtyeight.com/congress-model-2018/house_district_forecast.csv")
+house_national_forecast <- read_csv("https://projects.fivethirtyeight.com/congress-model-2018/house_national_forecast.csv")
+
+colnames(house_district_forecast) <- colnames(house_district_forecast) %>% 
+  tolower() %>% 
+  str_replace_all(" ", "_")
+house_district_forecast <- house_district_forecast %>% 
+  mutate(state = as.factor(state), 
+         district = as.factor(district), 
+         candidate = as.factor(candidate),
+         party = as.factor(party), 
+         incumbent = as.logical(incumbent), 
+         model = as.factor(model)) %>% 
+  select(-special)
+
+colnames(house_national_forecast) <- colnames(house_national_forecast) %>% 
+  tolower() %>% 
+  str_replace_all(" ", "_")
+house_national_forecast <- house_national_forecast %>% 
+  mutate(
+    party = as.factor (party), 
+    model = as.factor (model)) %>% 
+  select(-state)
+
+devtools::use_data(house_district_forecast, overwrite = TRUE)
+devtools::use_data(house_national_forecast, overwrite = TRUE)
+
+# mueller-polls ---------------------------------------------------------------
+mueller_approval_polls <- read_csv("data-raw/mueller-polls/mueller-approval-polls.csv")
+colnames(mueller_approval_polls) <- colnames(mueller_approval_polls) %>% 
+  tolower() %>% 
+  str_replace_all(" ", "_")
+
+mueller_approval_polls <- mueller_approval_polls %>% 
+  mutate(
+    start = as.Date(start, "%m/%d/%y"), 
+    end = as.Date(end, "%m/%d/%y"), 
+    pollster = as.factor(pollster), 
+    population = as.factor(population), 
+    text = as.factor(text)
+  )
+devtools::use_data(mueller_approval_polls, overwrite = TRUE)
+
+# ncaa-womens-basketball-tournament ---------------------------------------------------------------
+ncaa_womens_basketball_tournament_history <- read_csv("data-raw/ncaa-womens-basketball-tournament/ncaa-womens-basketball-tournament-history.csv") %>% 
+  clean_names() 
+
+colnames(ncaa_womens_basketball_tournament_history) <- colnames(ncaa_womens_basketball_tournament_history) %>% 
+  str_replace_all("x", "")
+
+ncaa_womens_basketball_tournament_history <- ncaa_womens_basketball_tournament_history %>% 
+  mutate(
+    school = as.factor(school), 
+    seed = as.numeric(seed),
+    conference = as.factor(conference), 
+    conf_w = as.numeric(conf_w), 
+    conf_l = as.numeric(conf_l), 
+    conf_percent = as.numeric(conf_percent),
+    conf_place = as.factor(conf_place), 
+    how_qual = as.factor(how_qual), 
+    `1st_game_at_home`= as.factor(str_replace_all(`1st_game_at_home`, "\\^", "")),
+    tourney_w = as.logical(tourney_w), 
+    tourney_l = as.logical(tourney_l), 
+    tourney_finish = as.factor(tourney_finish), 
```

Comment:
could you order the levels for `tourney_finish` as follows? i.e. from losing earliest possible to winning the championship?
```
c("OR",  "1st", "2nd", "RSF",  "RF", "NSF", "N2nd", "Champ")
```